_data/CONTRIBUTORS.yml

```diff
@@ -100,3 +100,9 @@ Jan Küster:
     git: jankapunkt
     affiliation: Universität Bremen
     orcid: 0009-0008-8088-9837
+Kenneth Rioja:
+   git: kennethrioja
+   email: kenneth.brian.rioja@cern.ch
+   orcid: 0000-0002-5177-0019
+   role: editor
```

Comment:
Hi, the role "editor" is reserved for members of the editorial board, so I suggest to remove this for the time being @kennethrioja. The fix is to remove the role line please @kennethrioja.